### PR TITLE
[MU4] Fix #10028: Notehead popup design improvements

### LINKED ISSUE #10028: [MU4 Task] Notehead popup design improvements
[Tasks updated 14 Dec '21]

**Task description**
This task comprises a few minor UX improvements to the notehead popup in **Properties**. 
Importantly, it reinstates the "small notehead" functionality from MS3.6, which has been accidentally omitted from the MS4 project.

Tasks for implementation are:

1. Reposition "Hide notehead" checkbox beneath "Notehead type" panel
2. Implement "Notehead parentheses" toggle buttons (toggling functionality of notehead parentheses palette item)
3. Reinstate checkbox from MS3.6 (Inspector > Note > Small) and rename label to "Small notehead"
4. Rename "Dotted note position" label to "Duration note position"

NEW NOTEHEAD POPUP DESIGN_v2

The "small notehead" functionality to be reinstated from MS3.6 **(TASK 2)** is this:
small-notehead-3 6-2

Link to Figma design file: https://www.figma.com/file/l0TQUrNFZmEfrZnfHhFDfb/Musescore-Notation-App-MASTER?node-id=20835%3A668717

**Note also**
Issue #9169 includes a bug fix for this panel, as well as a task for disabling the "Duration dot position" buttons depending on what is selected in the score. Please advise if these tasks require further illustration and I'll be happy to supply it. 

Let me know if anything is unclear, or if you have any questions!
Thanks 🙂

### PR DESCRIPTION
**Pull request by RomanPudashkin**
Resolves: https://github.com/musescore/MuseScore/issues/10028

### CODE REVIEW

**Comment by bkunda** [comment]
One small fix please before merging! 

"Duration **note** position" should read "Duration **dot** position" (bold used for emphasis here only)

Screenshot 2022-03-31 at 11 20 55 pm

Everything else seems to work fine on both Win and Mac, so happy for this to be merged once this label has been corrected.

Thanks!

File: src/inspector/view/qml/MuseScore/Inspector/notation/notes/HeadSettings.qml [hunk 1]
```diff
@@ -45,31 +45,61 @@ FocusableItem {
...
+            propertyItem: root.model ? root.model.isHeadSmall : null
+
+            navigation.name: "SmallNoteHeadBox"
+            navigation.panel: root.navigationPanel
+            navigation.row: hideNoteheadCheckBox.navigation.row + 1
         }
 
         FlatRadioButtonGroupPropertyView {
-            id: dottedNotePositionSection
-            titleText: qsTrc("inspector", "Duration dot position")
```
[the remark this answers is not in the document]
Comment (Jojo-Schmitz) [reply src/inspector/view/qml/MuseScore/Inspector/notation/notes/HeadSettings.qml:67]: Isn't it rather "Augmentation dot position"?

[the remark this answers is not in the document]
Comment (bkunda) [reply src/inspector/view/qml/MuseScore/Inspector/notation/notes/HeadSettings.qml:67]: Well, "jein"....

There was discussion about this in #10028, then @Tantacrul ran a poll and there seemed to be a strong preference for "Duration dot"

Screenshot 2022-03-31 at 11 31 29 pm

Either choice will have its detractors, and I find merit with both options. So in this case I'm happy to go with the voice of the majority opinion. 🙂

[the remark this answers is not in the document]
Comment (Jojo-Schmitz) [reply src/inspector/view/qml/MuseScore/Inspector/notation/notes/HeadSettings.qml:67]: Ah, yes... But a IMHO good reason why public polls are not always a good choice...

**Review by Eism — APPROVED** [review]

**Pull request merged**